ci-operator/config/openshift/azure-disk-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master.yaml

```diff
@@ -3,11 +3,23 @@ base_images:
     name: "4.8"
     namespace: ocp
     tag: base
+  ocp_builder_rhel-8-golang-1.15-openshift-4.8:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.8
```

Comment:
I would expect us to use go 1.16 instead

Reply:
BTW, note that go 1.16 build tools were added recently, so the Dockerfiles should probably be updated as well

Reply:
This has been updated for both the driver and operator. I've also opened PRs for both of these:

* Driver - https://github.com/openshift/azure-disk-csi-driver/pull/4
* Operator - https://github.com/openshift/azure-disk-csi-driver-operator/pull/6